.changeset/nine-vans-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': minor
+---
+
+**feat**(`STextField`): add `prefix` slot - inline elements before input and `filledState` prop to use filled state when prefix presents
```

Comment:
```suggestion
**feat**(`STextField`): add `prefix` slot to render elements before input, and `filled-state` prop to manually activate the filled state on the component when the prefix presents
```

Probably put `filled-state` in a separate changeset?

Reply:
changed
`filled-state` is mostly part of prefix feature